src/wp-admin/options.php

```diff
@@ -217,11 +227,11 @@
 		check_admin_referer( $option_page . '-options' );
 	}
 
-	if ( ! isset( $whitelist_options[ $option_page ] ) ) {
+	if ( ! isset( $allowed_options[ $option_page ] ) ) {
 		wp_die(
 			sprintf(
 				/* translators: %s: The options page name. */
-				__( 'Options page %s not found in the options whitelist.' ),
+				__( 'Options page %s not found in the options allowed list.' ),
```

Comment:
```suggestion
				__( 'Options page %s not found in the allowed options list.' ),
```

I think that might be read better.